test/test_deployment.rb

```diff
@@ -0,0 +1,25 @@
+require 'test_helper'
+
+# Replication Controller entity tests
+class TestDeployment < MiniTest::Test
+  def test_get_from_json_v1
+    stub_request(:get, %r{/deployment})
+      .to_return(body: open_test_file('deployment.json'),
+                 status: 200)
+
+    Kubeclient::Deployment.new
+    client = Kubeclient::Client.new 'http://localhost:8080/apis', 'extensions/v1beta1'
```

Comment:
@simon3z , yup, I didn't want to mess with the current code too much, and I figured the test should change anyway when 'Deployments' gets merged to the official api, but I'm up for suggestions.